teamcity-common/src/main/java/com/cyberark/common/HttpClient.java

```diff
@@ -65,7 +66,7 @@ public static HttpResponse request(String urlString, String method, String authH
             URL url = new URL(urlString);
             HttpsURLConnection conn = (HttpsURLConnection) url.openConnection();
 
-            if (certificateContent != null) {
+            if (certificateContent != null && !certificateContent.trim().isEmpty()) {
```

Comment:
Nice! Could there be a test case to accompany this change ? Just to ensure that we automate the detection of regressions

Reply:
Right now I'm trying to fix the bug as it's a showstopper. The maintainer of the repo/plugin should probably advise what the tests should look like - i can't see any so far.

Reply:
You're right, no need to add tests since there aren't any. Will have to be manual testing for the time being since we have not had time to bring this project into our maintained ecosystem.